proto/configpb.proto

```diff
@@ -60,8 +53,8 @@ message ConfigEntry {
 message CreateRequest {
     uint64 cluster_id = 1;
     Version version = 2;
-    bytes component_id = 3;
-    Component component = 4;
+    string component = 3;
```

Comment:
I think we could keep in consistent with TiDB system table TIDB_CLUTER_INFO, use TYPE and ADDRESS to represent a cluster component node, which maybe make the user have a more understanding.

Reply:
It's not only for tidb but also for other services, I prefer to keep it generic.